AbandonedCartReminder.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/*************************************************************************************/
+/*      This file is part of the Thelia package.                                     */
+/*                                                                                   */
+/*      Copyright (c) OpenStudio                                                     */
+/*      email : dev@thelia.net                                                       */
+/*      web : http://www.thelia.net                                                  */
+/*                                                                                   */
+/*      For the full copyright and license information, please view the LICENSE.txt  */
+/*      file that was distributed with this source code.                             */
+/*************************************************************************************/
+
+namespace AbandonedCartReminder;
+
+use Propel\Runtime\Connection\ConnectionInterface;
+use Thelia\Core\Translation\Translator;
+use Thelia\Install\Database;
+use Thelia\Model\Base\MessageQuery;
+use Thelia\Model\Lang;
+use Thelia\Model\LangQuery;
+use Thelia\Model\Message;
+use Thelia\Module\BaseModule;
+
+class AbandonedCartReminder extends BaseModule
+{
+    /** @var string */
+    const DOMAIN_NAME = 'abandonedcartreminder';
+
+    const VAR_DELAI_RAPPEL_1 = 'delai_rappel_1_en_minutes';
```

Comment:
This is not very flexible, it's better to use 'n' instead of "1","2" and allow user to add much reminder as they want

Reply:
It's indeed a good idea but it will need lots of modifications